Resources/ServerInfo/_Impstation/Guidebook/Mobs/Gray.xml

```diff
@@ -1,42 +1,56 @@
 <Document>
-  # Gray
-
+  # Grays
   <Box>
-    <GuideEntityEmbed Entity="MobGray" Caption=""/>
+  <GuideEntityEmbed Entity="MobGray" Caption=""/>
   </Box>
+  ## Physiology
+  Grays are much like Humans, in many ways. Their basic external physiology is largely similar to that of Humans, although they exhibit no sexual dimorphism. Internally, however, Grays are remarkably different - The structures of their organs are utterly unique among observed species, and [color=#ffa500]their blood is composed entirely of liquid Radium, which cannot be metabolized from any known source.[/color] 
+  
+  Their bodies are adapted for exposure to extreme environments, so they take [color=#1e90ff]60% less damage from Cold and Heat, and 80% less Poison damage,[/color] but are otherwise very fragile - They take [color=#ffa500]50% more Blunt, Slash, and Piercing damage.[/color]
+  
+  ## Language
+  [color=#ffa500]The Gray language, colloquially known as "Grayspeak," has evaded attempts at automatic translation up to this point.[/color] Although some Grays have, through various means, learned how to speak Common, [color=#ffa500]most Grays are largely unintelligible by other species.[/color]
+  
+  ## Culture
+  Grays are cultural sponges, and appear to absorb elements of other Humanoid cultures rapidly through exposure. However, they have been observed to possess a singular fascination with Human culture from the early Internet era, and their knowledge of it appears to terminate around the year 2010. It is unknown precisely why this is the case.
+
+  ## GRAY NAMING CONVENTIONS
+
+  Grays typically name themselves after Grayspeak-adjacent sounds, or "corrupted," Grayspeak-ified versions of things that interest them from other Humanoid cultures - generally surface-level interpretations of the names of products. For example, "Tonka Truck"  may become "Tonky Chuck." Try to avoid outright, unchanged references to recent media or memes.
+
+  - [color=#449944]GOOD:[/color] Carp Shitto
+  - [color=#449944]GOOD:[/color] Gnarpy Da
+  - [color=#449944]GOOD:[/color] Yippi Zoota
+  - [color=#449944]GOOD:[/color] Tonky Chuck
+  - [color=#449944]GOOD:[/color] New Ports
+  - [color=#449944]GOOD:[/color] Ford Mustang
+  - [color=#449944]GOOD:[/color] Images
+  - [color=#449944]GOOD:[/color] Emess Paint
+  - [color=#449944]GOOD:[/color] Spunch Bop
+  - [color=#994444]BAD:[/color] Isuggest Forcefem
```

Comment:
this one should be under GOOD though

Reply:
isuggest forcefem? it's been under BAD since the introduction of the naming conventions entries. it's an example of cuck's law. it's funny enough to not get admin renamed, but shouldn't be emulated or taken as an example of a good name

Reply:
the only reason Isuggest Forcefem wasn't admin renamed is because the admin misclicked and ended the round instead

Reply:
lmao